### ISSUE TITLE
Undefined variable: address

### ISSUE THREAD

**Issue by daveherman71** [Textalk/websocket-php#136]
If you have set a stream context with the **persistent** constructor option to **true** then PHP throws the following error:
```
A PHP Error was encountered

Severity: Notice
Message:  Undefined variable: address
Filename: lib/Client.php
Line Number: 202
```

At this point $address is not defined because it is set within the `if (!$persistent || ftell($this->socket) == 0) {` statement starting at line 131 which cannot be reached.

The client was created as follow:
```
$context = stream_context_create();

stream_context_set_option($context, 'ssl', 'verify_peer', FALSE);
stream_context_set_option($context, 'ssl', 'verify_peer_name', FALSE);

$client = new WebSocket\Client("ws://echo.websocket.org/", array(
	'context' => $context,
	'timeout' => 300
));
```
A proposed fix would be to move line 202 (`$this->logger->info("Client connected to {$address}");`) into the `if (!$persistent || ftell($this->socket) == 0) {` statement.

**Comment by sirn-se** [comment]
PR to fix this issue is pending; https://github.com/Textalk/websocket-php/pull/137

**Comment by sirn-se** [comment]
Included in patch version `1.5.3`.
